themes/michelebologna.zsh-theme

```diff
@@ -68,8 +68,14 @@ ZSH_THEME_GIT_PROMPT_AHEAD_REMOTE=">"
 ZSH_THEME_GIT_PROMPT_BEHIND_REMOTE="<"
 ZSH_THEME_GIT_PROMPT_DIVERGED_REMOTE="$red<>"
 
+ZSH_THEME_HG_PROMPT_MODIFIED="$red*"
+ZSH_THEME_HG_PROMPT_ADDED="$green+"
+ZSH_THEME_HG_PROMPT_UNTRACKED="$blue%%"
+
 PROMPT='$username_output$hostname_output:$current_dir_output%1(j. [$jobs_bg].)'
 GIT_PROMPT='$(out=$(git_prompt_info)$(git_prompt_status)$(git_remote_status);if [[ -n $out ]]; then printf %s " $white($green$out$white)$reset";fi)'
 PROMPT+="$GIT_PROMPT"
+HG_PROMPT='$(out=$(hg_prompt_info);if [[ -n $out ]]; then printf %s " $white($green$out$white)$reset";fi)'
```

Comment:
I'm sure there's a cleaner way to go about this; and the same with `$GIT_PROMPT`. Have a look at [conditional substrings in prompts](http://zsh.sourceforge.net/Doc/Release/Prompt-Expansion.html#Conditional-Substrings-in-Prompts).